src/link/internal/base.ts

```diff
@@ -31,7 +31,7 @@ export const createBaseLink = <TRouter extends AnyRouter>(
             if (id !== trpc.id) return;
 
             if ('error' in trpc) {
-              const error = runtime.transformer.deserialize(trpc.error);
+              const error = trpc.error;
```

Comment:
Do we need to destructure this? I think we can pass it directly to the line below.